mne/decoding/transformer.py

```diff
@@ -61,29 +93,60 @@ def fit(self, epochs_data, y):
         self : instance of Scaler
             Returns the modified instance.
         """
+        from sklearn.base import clone
         if not isinstance(epochs_data, np.ndarray):
             raise ValueError("epochs_data should be of type ndarray (got %s)."
                              % type(epochs_data))
 
         X = np.atleast_3d(epochs_data)
+        n_epochs, n_channels, n_times = X.shape
+        X = X.transpose(1, 0, 2)
+        self.ch_mean_ = np.zeros(n_channels)
+        self.scale_ = np.zeros(n_channels)
+        self.ch_median_ = np.zeros(n_channels)
+
+        if self.scalings == 'standard':
+            from sklearn.preprocessing import StandardScaler
+            std_scaler = StandardScaler(self.with_mean, self.with_std)
+            self.scalers_ = list()
+            X = X.reshape(n_channels, -1)
+            for _, pick in self._pick_types:
+                scaler = clone(std_scaler)
+                scaler.fit(X[pick].T)
+                self.scalers_.append(scaler)
+
+                # StandardScaler scale_ attribute introduced in version 0.18
+                if hasattr(scaler, "scale_"):
+                    self.ch_mean_[pick], self.scale_[pick] = (
+                        scaler.mean_, scaler.scale_)
+                else:
+                    self.ch_mean_[pick], self.scale_[pick] = (
```

Comment:
Again, the same reason as above, usage for the users after scaling.. If not required I will remove all the three attributes.